Update troubleshooting docs for `pulumi refresh` changes (#1915)
pulumi/pulumi-hugo: themes/default/content/docs/support/troubleshooting.md

## Recovering from an Interrupted Update {#interrupted-update-recovery}

> If the Pulumi CLI is interrupted when performing a deployment, you may see a warning message

Reviewer: why is this a`may` and not a `will`?

Author: You will only see this message if the interrupt interrupts an operation. If it is between updates (just starting, ending, or literally between updates) you won't get the warning, because no operation was interrupted.

If `pulumi cancel` fails with `error: [400] Bad Request: the update has already completed`, you can safely ignore
that error and continue with the next step.

> You should then run `pulumi refresh`. This will remove any pending operations cleanly,
> walking you through resolving any pending operations that Pulumi cannot fix on its own.

Reviewer: ```suggestion
Then run `pulumi refresh` to remove pending operations cleanly,
allowing you to resolve pending operations that Pulumi could not fix.
```
